src/button.tsx

```diff
@@ -172,7 +196,7 @@ const Button = (
         id='prefix-span'
         sx={{ display: 'inline-block', ...prefixOffset }}
       >
-        {prefix && prefix}
+        {clonedPrefix && clonedPrefix}
```

Comment:
I know the change here is just a variable name update, but it seems like the `&&` is not doing anything?
```suggestion
        {clonedPrefix}
```